public/css/main.css

```diff
@@ -0,0 +1,934 @@
+/*&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+    General styles
+
+&&&&&&&&&&&&&&&&&&&&&&&&&&&*/
+html {
+  font-family: Helvetica, sans-serif;
+  font-size: 1em;
```

Comment:
`1em` at the `html` level is the same as `100%` which is the same as the default value, so no need to specify it here.